packages/ui/src/components/modals/EditNames.tsx

```diff
@@ -130,4 +139,10 @@ export default styled(EditNames)`
     margin-bottom: 1rem;
     align-items: end;
   }
+
+  .closeButton {
+    position: absolute;
+    right: 0.5rem;
+    top: 0.5rem;
```

Comment:
We are getting rid of classes in our code. Please use the styled component to apply the styles (https://emotion.sh/docs/styled). As you are using the same icon and styles 3 times, you might create a component for that, f.e in src/components/library/CloseButton.tsx